src/vs/workbench/browser/actions/quickAccessActions.ts

```diff
@@ -147,10 +151,33 @@ registerAction2(class QuickAccessAction extends Action2 {
 
 	run(accessor: ServicesAccessor, prefix: undefined): void {
 		const quickInputService = accessor.get(IQuickInputService);
-		quickInputService.quickAccess.show(typeof prefix === 'string' ? prefix : undefined, { preserveValue: typeof prefix === 'string' /* preserve as is if provided */ });
+		const searchText = getFileSearchText(accessor) ?? '';
+		quickInputService.quickAccess.show(typeof prefix === 'string' ? prefix : searchText, { preserveValue: typeof prefix === 'string' /* preserve as is if provided */ });
 	}
 });
 
+
+function getFileSearchText(accessor: ServicesAccessor): string | '' {
```

Comment:
```suggestion
function getFileSearchText(accessor: ServicesAccessor): string {
```
it's obvious from the body that empty string is returned

Reply:
Done!